src/status_im2/common/standard_authentication/standard_auth/view.cljs

```diff
@@ -1,14 +1,14 @@
 (ns status-im2.common.standard-authentication.standard-auth.view
   (:require
-    [quo2.core :as quo]
-    [quo2.theme :as quo.theme]
-    [reagent.core :as reagent]
-    [utils.re-frame :as rf]
-    [react-native.touch-id :as biometric]
-    [utils.i18n :as i18n]
-    [taoensso.timbre :as log]
-    [status-im2.common.standard-authentication.enter-password.view :as enter-password]
-    [react-native.core :as rn]))
+   [quo2.core :as quo]
+   [quo2.theme :as quo.theme]
+   [reagent.core :as reagent]
+   [utils.re-frame :as rf]
+   [react-native.touch-id :as biometric]
+   [utils.i18n :as i18n]
+   [taoensso.timbre :as log]
+   [status-im2.common.standard-authentication.enter-password.view :as enter-password]
+   [react-native.core :as rn]))
```

Comment:
I think @J-Son89 fixed the formatting here, but our `lint-fix` tool will revert this change.
@ilmotta do we have a way to fix this? maybe in this PR?

Reply:
@ulisesmac, this issue seems to be related to how `clojure-lsp` cleans up namespaces. It aligns the opening square bracket `[` with the colon in `:require`. The way we configured zprint causes it to align the namespaces with the letter `r`.

When I run `make lint-fix` zprint aligns the namespaces to `r` in `:require`, which seems to be correct.

Reply:
> When I run make lint-fix zprint aligns the namespaces to r in :require, which seems to be correct.

I think this behavior is not correct. it is a list, not a function call:
```clojure
;; list:
(:require
 :b 
 :c
 [])
 
;; fn call
(require
  :b 
  :c
  []) 
```

Reply:
apolgies, it's probably my local formatter but not the linter. I'll check

Reply:
> I think this behavior is not correct. it is a list, not a function call:

I meant correct from zprint's point of view. But I agree how clojure-lsp formats is the correct way. @yqrashawn has the greatest knowledge about zprint.

@yqrashawn, do you know if we can tell zprint to format namespaces identically to clojure-lsp? At the moment, `clojure-lsp clean-ns` command has one less whitespace than zprint.

Reply:
Will look into this maybe later next week https://github.com/status-im/status-mobile/issues/17641